### ISSUE TITLE
Invalid image URL

### ISSUE THREAD

**Issue by erickotsuka** [ThundeRatz/labelling-web-app#6]
Ao acessar o web app [aqui](https://cone-labelling.herokuapp.com/label), as imagens deixaram de aparecer, aparecendo "Invalid image URL" sempre, mesmo após atualizar a página. (Ao atualizar a página, o ID muda).
Copiei um [URL que aparecia](https://f001.backblazeb2.com/b2api/v1/b2_download_file_by_id?fileId=4_z8126b7d060d92d74599e0618_f1021bf36588d327c_d20161223_m010407_c001_v0001032_t0054). Ao abrí-lo, um arquivo chamado b2_download_file_by_id foi baixado. Eu o abri e o conteúdo era esse:
```
[net]
batch=64
subdivisions=16
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
max_batches = 40100
policy=steps
steps=-1,100,20000,30000
scales=.1,10,.1,.1

[convolutional]
batch_normalize=1
filters=16
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=1

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

###########

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=1024
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=30
activation=linear

[region]
anchors = 1.08,1.19,  3.42,4.41,  6.63,11.38,  9.42,5.11,  16.62,10.52
bias_match=1
classes=1
coords=4
num=5
softmax=1
jitter=.2
rescore=1

object_scale=5
noobject_scale=1
class_scale=1
coord_scale=1

absolute=1
thresh = .6
random=1
```

**Comment by tiagoshibata** [comment]
OK, foi cagada minha adicionando as imagens do dataset antigo no banco de dados, foram uns arquivos a mais que não eram imagens (incluíndo a descrição da rede). Valeu por buscar o erro, arrumo mais tarde.

**Comment by erickotsuka** [comment]
Parece que já arrumou. Vou fechar a issue.
